src/ceph-daemon/ceph-daemon

```diff
@@ -1698,6 +1712,25 @@ def _get_parser():
         required=True,
         help='daemon name (type.id)')
 
+    parser_logs = subparsers.add_parser(
+        'logs', help='fetch the log for a daemon\'s container')
+    parser_logs.set_defaults(func=command_logs)
+    parser_logs.add_argument(
+        '--fsid',
+        required=True,
+        help='cluster FSID')
+    parser_logs.add_argument(
+        '--name', '-n',
+        required=True,
+        help='daemon name (type.id)')
+    parser_logs.add_argument(
+        '-f', '--follow',
+        action='store_true',
+        help='Follow log output')
+    parser_logs.add_argument(
+        '-t', '--tail',
```

Comment:
Remove `-t` and allow `--tail` only?

This could be (semi-) confusing if we decided to add the podman timestamp option ...
```
      --tail uint      Output the specified number of LINES at the end of the logs.  Defaults to 0, which prints all lines
  -t, --timestamps     Output the timestamps in the log

```

Reply:
good call, fixed